### ISSUE TITLE
Add `version: latest` to automatically download the latest version

### ISSUE THREAD

**Issue by Kerumen** [phrase/setup-cli#38]
Similar to the [Expo GitHub Action](https://github.com/expo/expo-github-action), add a new string `latest` for the version parameter to avoid specifying a version number and ensure the CLI is always updated.

**Comment by Kerumen** [comment]
Yes that's correct :) To avoid having to update all of our workflows everytime a new version is published.

**Comment by jablan** [comment]
Makes sense. I'll open an internal ticket for that, it's a small change so I expect it to be picked up soon.

**Comment by tkrotoff** [comment]
+1

We have the same issue, Phrase CLI asks to be updated regularly in our CI:
> Please consider updating the Phrase Strings CLI client (2.31.2 < 2.32.0)
> You can get the latest version from https://phrase.com/cli.

We would like to be able to set
```
with:
  version: latest
```
(should be the default if no version is specified IMHO)

**Comment by jablan** [comment]
hey @tkrotoff @Kerumen seems that the change necessary is bigger than initially assumed, therefore we couldn't fit it in yet. would it in the meantime help to optionally disable the warning message?

**Comment by tkrotoff** [comment]
@jablan 

> optionally disable the warning message?

I would keep the warning message: this is not a big issue, it's just boring
